src/basic/fs-util.c

```diff
@@ -38,6 +38,14 @@
 #include "umask-util.h"
 #include "user-util.h"
 
+#if defined(O_SEARCH) && defined(O_PATH) && O_SEARCH == O_PATH
```

Comment:
Please drop this block. I guess it is not necessary anymore. Right?

Reply:
Thanks for catching this, I missed removing it.